examples/lcd/src/main.c

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (C) 2019, Sergey Shcherbakov
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Example description:
+ *
+ * Printing TICK timer counter value on a HD44780 display
+ */
+
+// Std headers
+#include <stddef.h>
+
+// libopencm3 headers
+#include <libopencm3/cm3/systick.h>
+#include <libopencm3/cm3/cortex.h>
+#include <libopencm3/stm32/gpio.h>
+#include <libopencm3/stm32/rcc.h>
+
+#include "../../libopencm3-drivers/include/hd44780.h"
+
+// Description of PINS that used as a bus for HD44780 display
+static struct hd44780_bus lcd_bus = {
+	.rs = {GPIOE, GPIO7},
+	.e = {GPIOE, GPIO11},
+	.rnw = {GPIOE, GPIO10},
+	.db7 = {GPIOE, GPIO15},
+	.db6 = {GPIOE, GPIO14},
+	.db5 = {GPIOE, GPIO13},
+	.db4 = {GPIOE, GPIO12},
+};
+
+// Variable that used for counting ms
+static volatile uint32_t tick_ms_count;
+
+// Basic sleep function that uses TICK and busy loop
+void sleep_ms(uint32_t ms)
+{
+	uint32_t current_ms = tick_ms_count;
+	while ((tick_ms_count - current_ms) < ms) {
```

Comment:
Could be a bit shorter?
while ((tick_ms_count - current_ms) < ms);

Reply:
Agree. I'll change it.